vscode/README.md

```diff
@@ -29,6 +29,29 @@ See complete information about features [here](https://shopify.github.io/ruby-ls
 If you experience issues, please see the [troubleshooting
 guide](https://github.com/Shopify/ruby-lsp/blob/main/TROUBLESHOOTING.md).
 
+### Copilot chat agent
+
+For users of Copilot, the Ruby LSP contributes a Ruby agent for AI assisted development of Ruby applications. Below you
+can find the documentation of each command for the Ruby agent. For information about how to interact with Copilot Chat,
+check [VS Code's official documentation](https://code.visualstudio.com/docs/copilot/copilot-chat).
+
+#### Design command
+
+The `@ruby /design` command is intended to be a domain driven design expert to help users model concepts for their
+applications. Users should describe what type of application they are building and which concept they are trying to
+model. The command will read their Rails application's schema and use their prompt, previous interactions and the schema
+information to provide suggestions of how to design the application. For example,
+
+```
+@ruby /design I'm working on a web application for schools. How do I model courses? And how do they relate to students?
+```
+
+The output is a suggested schema for courses including relationships with users. In the chat window, two buttons will appear: `Generate with Rails`, which invokes the Rails generators to create the models suggested, and `Revert previous generation`, which will delete files generated by a previous click in the generate button.
+
+As with most LLM chat functionality, suggestions may not be fully accurate, especially in the first iteration. Users can
+continue chatting with the `@ruby` agent to fine tune the suggestions given, before deciding to move forward with
+generation.
+
```

Comment:
Is it worth pointing people to the DX Slack for feedback or questions, or is the general link is enough?

Reply:
Yeah, good idea!